logproof/src/math.rs

```diff
@@ -653,6 +653,49 @@ where
     }
 }
 
+/**
+ * Computes the quotient and remainder of a / b.
+ */
+pub fn div_rem_bigint<const N: usize>(a: BigInt<N>, b: BigInt<N>) -> (BigInt<N>, BigInt<N>) {
```

Comment:
We have multiple math libraries. Currently this is defined in the zkp crate; @rickwebiii what do you think about moving this to `sunscreen_math`? That dependency is already imported into this crate.

Reply:
I think we should mark this deprecated and use `https://docs.rs/crypto-bigint/latest/crypto_bigint/struct.Uint.html#method.div_rem`

Reply:
Done.